CardPayments/src/main/java/com/paypal/android/cardpayments/CardResult.kt

```diff
@@ -7,13 +7,23 @@ import android.net.Uri
  *
  * @property [orderId] associated order ID.
  * @property [liabilityShift] Liability shift value returned from 3DS verification
+ * @property [status] status of the order
+ * @property [didAttemptThreeDSecureAuthentication] 3DS verification was attempted.
+ * Use v2/checkout/orders/{orderId} in your server to get verification results.
  */
 data class CardResult(
     val orderId: String,
 
     /**
      * @suppress
      */
+    @Deprecated("Use status instead.")
     val deepLinkUrl: Uri? = null,
-    val liabilityShift: String? = null
+
+    @Deprecated("Use didAttemptThreeDSecureAuthentication instead.")
+    val liabilityShift: String? = null,
+
+    var status: String? = null,
```

Comment:
Basic q but is there a reason we use a `var` here and a `val` for all the other ones?

Reply:
That's a good callout. It would make more sense if it was `val` to me. I don't think there's a good reason for it to be a mutable property.

Reply:
Fix: [9271cab](https://github.com/paypal/paypal-android/pull/266/commits/9271cabeccc823ae12faac79e7447cc775e3cd61)